tests/conftest.py

```diff
@@ -16,6 +16,13 @@
 
 from .data import canned_responses
 
+try:
+    import flask
+
+    assert flask
```

Comment:
What does `assert flask` do?

Reply:
Oh, I did that on the fly for flake8's F401. Let me replace with a noqa comment.